stress/common/src/main/java/alluxio/stress/client/ClientIOSummary.java

```diff
@@ -177,29 +177,10 @@ public List<Graph> generate(List<? extends Summary> results) {
               summaries.stream().filter(x -> x.mParameters.mOperation == operation)
                   .filter(x -> x.mParameters.mReadRandom == readRandom)
                   .collect(Collectors.toList());
-
-          if (!opSummaries.isEmpty()) {
-            // first() is the list of common field names, second() is the list of unique field names
-            Pair<List<String>, List<String>> fieldNames = Parameters.partitionFieldNames(
-                opSummaries.stream().map(x -> x.mParameters).collect(Collectors.toList()));
-
-            // Split up common description into 100 character chunks, for the subtitle
-            List<String> subTitle = new ArrayList<>(Splitter.fixedLength(100).splitToList(
-                opSummaries.get(0).mParameters.getDescription(fieldNames.getFirst())));
-
-            for (ClientIOSummary summary : opSummaries) {
-              String series = summary.mParameters.getDescription(fieldNames.getSecond());
-              subTitle.add(series + ": " + DateFormat.getDateTimeInstance()
-                  .format(summary.getEndMs()));
-            }
-
-            for (ClientIOSummary summary: opSummaries) {
-              List<ClientIOTaskResult> clientResults = new ArrayList<>();
-              for (ClientIOTaskResult result: summary.getNodeResults().values()) {
-                clientResults.add(result);
-              }
-              graphs.addAll(clientResults.get(0).graphGenerator().generate(clientResults));
-            }
+          for (ClientIOSummary summary: opSummaries) {
+            List<ClientIOTaskResult> clientResults =
+                new ArrayList<>(summary.getNodeResults().values());
+            graphs.addAll(clientResults.get(0).graphGenerator().generate(clientResults));
```

Comment:
Since the input to the graph generator now contains results from multiple nodes, I wonder what the generated graph will look like? Does it multiply the number of diagrams by the number of nodes? That can be a bit verbose.

Reply:
No actually the data point of multiple clients are in one graph